Source/Dafny/Compilers/Compiler-python.cs

```diff
@@ -161,30 +165,15 @@ protected class ClassWriter : IClassWriter {
     }
 
     private ConcreteSyntaxTree CreateGetter(string name, Type resultType, IToken tok, bool isStatic, bool createBody,
-      ConcreteSyntaxTree methodWriter) {
-      throw new NotImplementedException();
+        ConcreteSyntaxTree methodWriter) {
+      return methodWriter.NewBlock($"def {name}():", open: BraceStyle.NewlineNoBrace, close: BraceStyle.Nothing);
     }
 
     private ConcreteSyntaxTree CreateMethod(Method m, List<TypeArgumentInstantiation> typeArgs, bool createBody,
       ConcreteSyntaxTree wr, bool forBodyInheritance, bool lookasideBody) {
-      if (!createBody) {
-        return null;
-      }
+      wr.Write($"def {IdName(m)}(");
 
-      var customReceiver = !forBodyInheritance && NeedsCustomReceiver(m);
```

Comment:
Did all of this disappear because `customReceiver` is never true in our test cases so far?

Reply:
Exactly.

Reply:
Great, I'm definitely in favour of removing code with no testing coverage (assuming it's not easy to add a test instead).